toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -526,7 +616,7 @@ func customizeOSContents(ctx context.Context, ic *ImageCustomizerParameters) err
 		previewFeatureEnabled := slices.Contains(ic.config.PreviewFeatures,
 			imagecustomizerapi.PreviewFeatureReinitializeVerity)
 		if !previewFeatureEnabled {
-			return fmt.Errorf("Please enable the '%s' preview feature to customize a verity enabled base image",
+			return fmt.Errorf("%w (feature='%s')", ErrVerityPreviewFeatureRequired,
```

Comment:
`imagecustomizerapi.PreviewFeatureReinitializeVerity` is a constant string. So, I would integrate the string into the `ErrVerityPreviewFeatureRequired` message.

Reply:
added the constants to the message